api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
```

Comment:
I think you can do a big `sed`

```suggestion
// Copyright 2016-2021 Datadog, Inc.
```